src/Workflow/MoveJatsAssets.php

```diff
@@ -101,9 +103,24 @@ private function processAsset(Element $asset, PutTask $task) : PromiseInterface
         return new Coroutine(
             function () use ($asset, $task) : iterable {
                 /** @var ResponseInterface $response */
-                $response = yield $this->client->requestAsync('GET', element_uri($asset));
+                $response = yield $this->client->requestAsync('GET', $uri = element_uri($asset));
+
+                try {
+                    $contentType = parse_media_type($response->getHeaderLine('Content-Type'));
+
+                    if ('octet-stream' === $contentType[1]) {
```

Comment:
any synonyms here (`binary`?) that should also be ignored?
The value itself is worth of a constant given its ubiquitousness.

Reply:
AFAIK `application/octet-stream` is official, but `binary/octet-stream` is commonly used. Will make more explicit.